python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -34,6 +35,26 @@ def __init__(self, config_file=None, context=None,
 
         self.api_instance = client.CustomObjectsApi()
 
+    def create_creds(self, platform, namespace=None, **kwargs):
```

Comment:
maybe it is not worth to make a single API with variable args since the argument list of the two are very different.

Reply:
@yuzisun You mean create two API for GCP and AWS? Personally I prefer to create unified API, since we may support other platforms such as Azure and IBM cloud etc... unified API for good usability.

Reply:
so if we support s3 creds, IBM Cloud storage should be supported by default?

Reply:
@animeshsingh Updated to S3, I think so. Thanks.

Reply:
> maybe it is not worth to make a single API with variable args since the argument list of the two are very different.

@yuzisun as we discussed offline, may I suggest to keep the unified API for easy maintaining and good UE. I updated the document to add column `Storage Type` to shows the arg belong to storage type (some of args for `All` while some for `S3 only`). If we separated API, need two maintain two or more table and some args may duplicate. Thanks a lot!